pyBumpHunter/bumphunter_2dim.py

```diff
@@ -636,6 +923,8 @@ def load_state(self, state: dict):
             self.t_ar = state["t_ar"]
         if "signal_eval" in state.keys():
             self.signal_eval = state["signal_eval"]
+        if "norm_scale" in state.keys():
```

Comment:
```suggestion
        if "norm_scale" in state:
```

That checks automatically in "keys"